.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - dev
+      - main
+    tags:
+  pull_request:
+  release:
+
+jobs:
+  check-copyright:
+    name: Copyright
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: License header
+        run: .scripts/check-copyright-notice.sh
+      - name: Authors
+        run: .scripts/check-notice-authors.sh origin/main
+
+  vet:
+    name: Vetting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Check vanity import
+        run: .scripts/check-vanity-imports.sh $GITHUB_WORKSPACE
+      - name: Lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.40.1
```

Comment:
latest version? If not, now is a good time to upgrade.

Reply:
updated to 1.43